src/components/Card.js

```diff
@@ -32,34 +53,29 @@ export default class Card {
       );
   }
 
-  _handleLikeButton() {
-    this._element
-      .querySelector(".card__like-button")
-      .classList.toggle("card__like-button_active");
-  }
-
-  _handleDeleteButton() {
+  handleDeleteButton() {
```

Comment:
Needs correction: Please rename this method to `deleteCard`, since it's not an event listener anymore